aiohue/sensors.py

```diff
@@ -426,6 +410,14 @@ def tholddark(self):
     def tholdoffset(self):
         return self.raw["config"]["tholdoffset"]
 
+    def process_update_event(self, update):
+        state = dict(self.state)
+
+        if "light" in update and update["light"]["light_level_valid"]:
+            state["lightlevel"] = update["light"]["light_level"]
```

Comment:
So we just had a bug where the temperature was in a different format than normal state. We fixes this in #65 

Are we sure this one is the same format?

Reply:
My bad, I assumed the value from the event was the same as the one in the state, but indeed the state is multiplied by 100.

I checked for the other events, which should be aligned between the state and the event:
- lightlevel: same value (integer in hue "light level" units)
- battery: same value (integer between 0-100)

So hopefully it should be fine for the rest.